package.json

```diff
@@ -5,8 +5,13 @@
   "main": "gulpfile.js",
   "scripts": {
     "build": "gulp",
-    "start": "node server.js"
+    "dev": "gulp dev",
+    "start": "node -e 'require(\"./server.js\")()'"
```

Comment:
`npm start` remains a way to serve `dist` in the use case that someone wants to host a mirror of C20 by checking out the repo, building it, then serving using the built-in server.